politeiawww/database/localdb/localdb.go

```diff
@@ -18,8 +19,31 @@ const (
 
 	UserVersion    uint32 = 1
 	UserVersionKey        = "userversion"
+
+	AccessTimePrefixKey = "accesstime-"
 )
 
+func getAccessTimeKey(email string) []byte {
+	return []byte(AccessTimePrefixKey + email)
+}
+
+// addAccessTime updates the user notifications by keeping it's length lower
+// or equal to the maximum number notifications per user.
+func addAccessTime(pat database.ProposalAccessTime, pats []database.ProposalAccessTime) []database.ProposalAccessTime {
```

Comment:
Each proposal should have only one timestamp associated with it. Hence, I think would be better to kill this function and use a map instead for storing the access Time. 
```Go
map[string]ProposalAccessTime // [token]ProposalAccessTime
````